galapagos_user_guide.md

```diff
@@ -370,25 +364,25 @@ winpty openssl pkcs12 -in <myclientkeystore.p12> -nocerts -nodes -out <myprivate
    Here you state why the topic should no longer be used and when it will be deprecated. Then click the yellow button to
    mark the respective topic as deprecated. All subscribers to this topic will be informed of the deactivation by email.
 
-   After a topic is marked as deprecated, this becomes visible on the surface. On the overview of the individual topic,
+   After a topic is marked as deprecated, it becomes visible on the surface. On the overview of the individual topic,
    a note appears next to the name of the topic that this topic can no longer be used in the near future:
    
    ![](.galapagos_user_guide_images/deprecation2.png)
    
-   In addition, under the "Browse Topics" tab, topics are marked with a line through the name of the topic:
+   In addition, under the "Browse Topics" tab, deprecated topics are crossed out:
```

Comment:
"topic crossed" hört sich auch nicht richtig an, denn nur der Name des Topics ist ja durchgestrichen, deshlab würde ich
"marked with a line through the name of the topic" reinnehmen.

Reply:
crossed out heißt durchgestrichen soweit ich weiß